docs/_static/niscope_usage.inc

```diff
@@ -7,11 +7,119 @@ The following is a basic example of using the **niscope** module to open a sessi
 
     import niscope
     with niscope.Session("Dev1") as session:
-        session.configure_vertical(range=1.0, coupling=niscope.VerticalCoupling.AC)
-        session.configure_horizontal_timing(min_sample_rate=50000000, min_num_pts=1000, ref_position=50.0, num_records=1, enforce_realtime=True)
+        session.channels[0].configure_vertical(range=1.0, coupling=niscope.VerticalCoupling.AC)
+        session.channels[1].configure_vertical(range=10.0, coupling=niscope.VerticalCoupling.DC)
+        session.configure_horizontal_timing(min_sample_rate=50000000, min_num_pts=1000, ref_position=50.0, num_records=5, enforce_realtime=True)
         with session.initiate():
-            wfm, wfm_infos = session[args.channels].fetch(num_samples=args.length)
-        print('Number of samples acquired: {:,}\n'.format(len(wfm)))
+            waveforms = session.channels[0,1].fetch(num_records=5)
+        for wfm in waveforms:
+            print('Channel {0}, record {1} samples acquired: {2:,}\n'.format(wfm.channel, wfm.record, len(wfm.wfm)))
+
+        # Find all channel 1 records (Note channel name is always a sting even if integers used in channel[])
+        chan1 = [wfm for wfm in waveforms if wfm.channel == '0']
+
+        # Find all record number 3
+        rec3 = [wfm for wfm in waveforms if wfm.record == 3]
+
+The waveform returned from `fetch <http://nimi-python.readthedocs.io/en/master/niscope/functions.html#niscope.Session.fetch>`_ is a flat list of Python objects
+
+    - Attributes:
+
+        -  **relative_initial_x** (float) the time (in seconds) from the trigger to the first sample in the fetched waveform
+        -  **absolute_initial_x** (float) timestamp (in seconds) of the first fetched sample. This timestamp is comparable between records and acquisitions; devices that do not support this parameter use 0 for this output.
+        -  **x_increment** (float) the time between points in the acquired waveform in seconds
+        -  **channel** (str) channel name this waveform was asquire from
+        -  **record** (int) record number of this waveform
+        -  **gain** (float) the gain factor of the given channel; useful for scaling binary data with the following formula:
+
+            .. math::
+
+                voltage = binary data * gain factor + offset
+
+        -  **offset** (float) the offset factor of the given channel; useful for scaling binary data with the following formula:
+
+            .. math::
+
+                voltage = binary data * gain factor + offset
+
+        - **wfm** (array of float) floating point array of samples. Length will be of the actual samples acquired
```

Comment:
not always an array, find more generic term?
not always float, right?

Reply:
Always an array. Always a float.

However, the `fetch_into()` one is not so that one changed to (numpy array of datatype used).